### DISCUSSION TITLE
2.3付过费，到现在显示仍是免费版，不能成为专业版， 使用不了。

### DISCUSSION THREAD

**Discussion by daisytan7980** [getlantern/lantern-forum-cn#243] [Q&A] [answered]
付费账单， 邮件发过很多次，   ,仍然没有给一个解决方宁。

**Comment by MaxMKMax** [comment]
@daisytan7980 谢谢您的反馈. 好的明白了! 我帮您和团队说明这个问题。我帮你紧急解决问题。非常抱歉影响了您的体验感。谢谢您的等待与支持。

**Reply by MaxMKMax** [reply]
@daisytan7980 您好! 已经帮您处理着了. 不好意思等等一下哦.

**Reply by raysir66** [reply]
您好，我也是这样。昨天付费了，一直看到的都是免费版
邮箱 
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@raysir66 好的. 已经提交上去. 请您等待负责团队处理. 谢谢您.

**Comment by daisytan7980** [comment]
收到了，非常快啊，

Mike K ***@***.***> 于2024年3月16日周六 01:20写道：

**Reply by MaxMKMax** [reply]
@daisytan7980 嗯嗯. 不客气的! 谢谢夸奖! 
谢谢您支持蓝灯.

**Reply by MaxMKMax** [reply]
@daisytan7980 您好。非常感谢您的称赞! 我们很高兴看到您满意。如果您满意我们的服务的话, 请前往应用商店 (Google Play Store/ IOS Play Store) 并给我们 5 星评级和评论。我们将非常感谢您！

**Reply by daisytan7980** [reply]
今年系统重装了， 电脑硬盘坏了，数据重装了，  系统重装了。 所以这个key丢失了，现在不上了。 我想升级了。

**Comment by daisytan7980** [comment]
我现在系统重装了，现在原来的key不能用了，怎么激活，我今年付过费了。The one-time issue regarding your Pro
account has been resolved. Because your account did not properly activate,
please utilize the following code to upgrade your account to Pro:
PY32R-KQC9Q-QJ96R-JPMQD-7YJ3Y [image:
c8394bbf8eecf7d3ee7a04806fd9f05.jpg][image:
image.png]

Mike K ***@***.***> 于2024年3月16日周六 01:20写道：

**Reply by daisytan7980** [reply]
你们查一下这个key , account,key , 也是你们发的邮件， 这个名 mytanxx， 后面是gmail

**Comment by daisytan7980** [comment]
我今年付过费了， 今年硬盘前几天坏了，重装了，现在用不了。 PY32R-KQC9Q-QJ96R-JPMQD-7YJ3Y，原来的key,用不了 
![微信图片_20250225220614](https://github.com/user-attachments/assets/72db42ec-e894-4144-8edb-f8a2ee670afc)

**Comment by daisytan7980** [comment]
能否发邮件，给我激活一下， 谢谢，我现在显示的， 就是免费的。 
![image](https://github.com/user-attachments/assets/d1c4d7eb-a340-4d24-8ba9-f2ebe61e0825)

**Comment by MaxMKMax** [comment]
@daisytan7980 您好, 请您放心. 我们会帮您解决问题. 请您提供我您的帐号邮箱. 谢谢.

**Reply by daisytan7980** [reply]
mytanxx@gmail.com,您好，这个是我的邮件， 此邮箱可以收到key 的信息，谢谢。

**Reply by daisytan7980** [reply]
@MaxMKMax  我发了贴子， 一直没有收到，今天 收到一个邮件，结果是苹果的安装版。 mytanxx@gmail.com

**Comment by MaxMKMax** [comment, marked as answer]
@daisytan7980 您好, 请您提供我您的帐号邮箱. 我才能帮您提交问题上去. 谢谢.

**Reply by daisytan7980** [reply]
mytanxx@gmail.com  以上是我的邮件， 谢谢。 因：电脑硬盘坏了， 操作系统重装，原来这个key,就用不了。

**Reply by daisytan7980** [reply]
何时能收到邮件

**Reply by daisytan7980** [reply]
@MaxMKMax 还没收到邮件，麻烦跟进一下， 谢谢

**Comment by daisytan7980** [comment]
还是没有收 你们的邮件

Mike K ***@***.***> 于2025年2月26日周三 23:25写道：

**Comment by daisytan7980** [comment]
此邮件就是我收到帐号的的邮件， 谢谢 ***@***.***

Mike K ***@***.***> 于2025年2月26日周三 23:25写道：

**Comment by keamy2025** [comment]
我这边问题解决了，邮件不用抄送过来了，谢谢！

 
发件人： daisytan7980
发送时间： 2025-03-03 14:19
收件人： getlantern/lantern-forum-cn
抄送： keamy2025; Comment
主题： Re: [getlantern/lantern-forum-cn] 2.3付过费，到现在显示仍是免费版，不能成为专业版， 使用不了。 (Discussion #243)
你们查一下这个key , account,key , 也是你们发的邮件， 这个名 mytanxx， 后面是gmail

**Reply by daisytan7980** [reply]
关键没有收到注册码的邮件的邮件，

**Reply by daisytan7980** [reply]
关键没有收到注册码的邮码， 快小半个月了。 不好意思，麻烦了。 谢谢。

**Comment by keamy2025** [comment]
麻烦邮件不要再抄送给我了，谢谢

 
发件人： daisytan7980
发送时间： 2025-03-05 14:40
收件人： getlantern/lantern-forum-cn
抄送： keamy2025; Comment
主题： Re: [getlantern/lantern-forum-cn] 2.3付过费，到现在显示仍是免费版，不能成为专业版， 使用不了。 (Discussion #243)
@MaxMKMax 我发了贴子， 一直没有收到，今天 收到一个邮件，结果是苹果的安装版。 ***@***.***

**Comment by daisytan7980** [comment]
1.电脑硬盘坏了，2.23 日操作系统win10重装了。 后面蓝灯的注册码注册以后，用不了。第二次缴费的订单号
***@***.***。去年（2024.2.3号付了,今年2.2付费）
，注册key : PY32R-KQC9Q-QJ96R-JPMQD-7YJ3Y

keamy2025 ***@***.***> 于2025年3月5日周三 14:42写道：
